contracts/test/utils/Permit.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "@openzeppelin/contracts/cryptography/ECDSA.sol";
+import "forge-std/Test.sol";
+import "contracts/Spender.sol";
+import "contracts/utils/SpenderLibEIP712.sol";
+import "contracts/utils/SignatureValidator.sol";
+import { getEIP712Hash } from "contracts-test/utils/Sig.sol";
+
+contract Permit is Test {
```

Comment:
From my understanding this permit util collection doesn't depend on the Test base contract, and also has no setup needs. 

Maybe it is enough to construct it as plain library to reduce inheritance.

Reply:
Originally wanted to put signSpendWithPermit() function in the sig.sol file,
but the vm.sign() function should still inherit from test.sol

Reply:
My bad 🤦‍♂️